CONTRIBUTING.md

```diff
@@ -0,0 +1,336 @@
+and provide the following information:
+
+- **Use a clear and descriptive title** for the issue to identify the suggestion.
+- **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+- **Provide specific examples to demonstrate the steps**.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+- **Explain why this enhancement would be useful** to most Terra users and isn't something that can or should be implemented as an addon.
+
+### Pull Requests
+
+This section guides you through submitting a bug report for Terra. Following these guidelines helps maintainers and the community understand
+your report, and spend their time fixing the issue instead of understanding what you mean.
+
+This section guides you through submitting an enhancement suggestion for Atom, including completely new features and minor improvements to
+existing functionality. Following these guidelines helps maintainers and the community understand your suggestion and find related
+suggestions.
+
+- Your pull request must be either on the latest version of Terra, or on a branch for a future release.
+- Make sure that there are no already existing PRs that fix this. If so, it will be closed as a duplicate.
+- Make sure that this change is actually within the scope of Terra and is something a terrain generator should be doing.
+- Make sure that this is not an issue with a specific Terra *pack*, and instead applies to all of Terra.
+- Make sure that you have filled out all the required information and given descriptions of everything.
+
+While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete
+additional design work, tests, or other changes before your pull request can be ultimately accepted.
+
+#### Before Submitting A Pull Request
+
+- You must be on the **LATEST** version of Terra to make sure your feature hasn't been added yet.
+- Search for any [already existing issues](https://github.com/PolyhedralDev/Terra/issues?q=is%3Aissue+) (Including closed!) with your
+  problem. If you open a duplicate, it will be closed as such.
+- Verify that this is actually within the scope of Terra.
+- Be sure that this is not a feature request that should be made for a specific Terra *pack*, and instead applies to all of Terra.
+- Be sure that this is not something that should be implemented as a Terra addon, and instead applies to all of Terra.
+- Make sure that you attach a copy of the latest.log file, if there are any exceptions thrown in the console. Putting *just* the
+  exception **is not enough**. We need to be able to check that there wasn't anything else before that caused it.
+
+#### How Do I Submit A (Good) Pull Request?
+
+Pull Requests are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an issue on our main repository and provide
```

Comment:
They're tracked as GitHub PR's

Reply:
copy and paste kicking me in the ass again